packages/mobile/src/screens/chat-screen/ChatListItemSkeleton.tsx

```diff
@@ -0,0 +1,68 @@
+import { View } from 'react-native'
+
+import Skeleton from 'app/components/skeleton'
+import { makeStyles } from 'app/styles'
+
+import { useStyles as useChatListItemStyles } from './ChatListItem'
```

Comment:
nit: Might make sense to move both components into a subfolder and put the styles in a `styles` file.